contracts/src/v0.8/workflow/test/WorkflowRegistryManager.t.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {Ownable2Step} from "../../shared/access/Ownable2Step.sol";
+import {WorkflowRegistry} from "../dev/WorkflowRegistry.sol";
+import {WorkflowRegistryManager} from "../dev/WorkflowRegistryManager.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract WorkflowRegistryManagerTest is Test {
```

Comment:
Please use the following test name format

```
test_Description for standard tests.
testFuzz_Description for fuzz tests.
test_RevertWhen_Condition for tests expecting a revert.
testFork_Description for tests that fork from a network.
```

and create a test file per function to be called. For the folder structure, see https://github.com/smartcontractkit/chainlink/pull/15165 as example.